1.md

```diff
@@ -0,0 +1,3 @@
+Как незарегистрируемый пользователь,  
```

Comment:
Are you sure this is acceptable Russian grammar?

Reply:
no i am not